src/cloud/utils/experiments.ts

```diff
@@ -0,0 +1,23 @@
+const pushDataLayerEvent = async (dataLayerEvent: string) => {
+  if (window.dataLayer) {
+    await window.dataLayer.push({event: dataLayerEvent})
+  }
+}
+
+const getVariant = (experimentID: string, attempts: number = 10): string => {
+  if (attempts <= 0) {
+    return ''
+  }
+  if (window['google_optimize']) {
+    return window['google_optimize'].get(experimentID) || null
```

Comment:
fyi, i changed this from `''` to `null` because `null` is slightly more explicit for how it's being used:

```ts
if (variantId) {
...
```